src/utils/node/utils.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import * as fsExtra from 'fs-extra';
+import * as os from 'os';
+import { NETWORK, OS } from '../constants';
+
+export const liskInstall = (installPath: string): string =>
+	installPath.replace('~', os.homedir);
+
+export const liskVersion = (version: string): string =>
+	`lisk-${version}-${os.type()}-x86_64`;
+
+export const liskTar = (version: string): string =>
+	`${liskVersion(version)}.tar.gz`;
+
+export const liskTarSHA256 = (version: string): string =>
+	`${liskTar(version)}.SHA256`;
+
+export const liskLatestUrl = (url: string, network: NETWORK) =>
+	`${url}/${network}/latest.txt`;
+
+export const liskSnapshotUrl = (url: string, network: NETWORK) =>
+	`${url}/${network}/blockchain.db.gz`;
+
+export const liskDbSnapshot = (networkName: string, network: NETWORK) =>
+	`${networkName}-${network}-blockchain.db.gz`;
+
+export const logsDir = (installPath: string) =>
+	`${liskInstall(installPath)}/logs`;
+
+export const SH_LOG_FILE = 'logs/lisk.out';
```

Comment:
why is this named `SH_LOG_FILE`? maybe `LOG_FILE_PATH`?

Reply:
this log file is basically for any shell command output, so it was `SH_LOG_FILE`, `LOG_FILE_PATH` will become generic?

Reply:
ah make sense =)